ref/pom.xml

```diff
@@ -17,6 +17,10 @@
             <groupId>${project.groupId}</groupId>
             <artifactId>smallrye-common-constraint</artifactId>
         </dependency>
+        <dependency>
+            <groupId>org.graalvm.sdk</groupId>
+            <artifactId>graal-sdk</artifactId>
+        </dependency>
```

Comment:
Not an expert, but this might be concerning for some consumer projects like WildFly, maybe?

Reply:
The dependency has `provided` scope. If it is not present in the run time, the exception-handling path should take over and `false` will be returned for `isBuildTime`.

The `smallrye-common-net` subproject also uses this dependency in `provided` scope.

Reply:
Ouch, I didn't look into the parent. Indeed `provided` scope makes sense.